docs/installation.md

```diff
@@ -198,14 +195,22 @@ If you are on a machine that is already set up to run Linux docker containers,
 please install [Docker Compose](https://docs.docker.com/compose/install/).
 If `docker-compose ps` runs without error, you can can go to step 2.
 
-#### Mac + Homebrew + Virtualbox quickstart
+#### Copy the `.python_env_SAMPLE` file over
 
-**Starting assumptions**: You already have homebrew and virtualbox installed.
+If this is your first time setting up the project, copy `.python_env_SAMPLE` to
+`.python_env`.
```

Comment:
Similarly to above, it'd be nice to include the explicit command for less-technical users.

```sh
$ cp .python_env_SAMPLE .python_env
```